src/VisualStudio/Core/Def/Implementation/AbstractOleCommandTarget.Execute.cs

```diff
@@ -20,6 +20,7 @@ internal abstract partial class AbstractOleCommandTarget
     {
         public virtual int Exec(ref Guid pguidCmdGroup, uint commandId, uint executeInformation, IntPtr pvaIn, IntPtr pvaOut)
         {
+            this.CommandID = commandId;
```

Comment:
I'm afraid this has to stay, because the editor handles this command and we get called via IVsTextViewFilter::GetPairExtents(). Its the same call the editor uses for GotoBrace and GotoBraceExt. So differentiate between the two, either the editor needs to make an API change to tell us or we cache the commandId in Exec.